python/tests/fixtures/predictor_fixtures.json

```diff
@@ -40,6 +40,9 @@
           "inference_logging": "ALL",
           "kafka_topic_dto": {
             "name": "topic"
+          },
+          "environment_dto": {
```

Comment:
Can you add asserts for `environment_dto` in the [predictor tests](https://github.com/logicalclocks/machine-learning-api/blob/954a07b9b2499f485e0d4678bb87b6c3e260a123/python/tests/test_predictor.py#L79)?

Reply:
added many test cases here